src/main/java/com/RouteController.java

```diff
@@ -96,7 +98,7 @@ public Route generateRoute(@RequestParam(value = "dest") int destID, @RequestPar
 
         if (startNode == null)
         {
-            return null; // Starting node could not be found within 300 feet of current location. Route not available.
+            return new Route(null, null, null, "A starting node could not be found for your current location."); // Starting node could not be found within 300 feet of current location. Route not available.
```

Comment:
I'm not sure this error message is really user friendly.

Something like "You are not close enough to campus or paths on campus to calculate a route using your current location"